### ISSUE TITLE
Select type of column from dropdown list

### ISSUE THREAD

**Issue by 1st** [jakob/Postico#113]
When I create new table I need to enter `TYPE` manually. It will be good to start typing and select value from list of suggestions. Sometimes I can forget what to type here - and I can see full list of available types and select on of them.

<img width="948" alt="screen shot 2015-11-03 at 17 15 03" src="https://cloud.githubusercontent.com/assets/630224/10911660/82c2bc14-824e-11e5-8bbb-84f04235a700.png">

Something like this, sit works in MySQL Workbench.

<img width="583" alt="screen shot 2015-11-03 at 17 16 21" src="https://cloud.githubusercontent.com/assets/630224/10911714/c4d02eb6-824e-11e5-9dcf-ff32f5096143.png">

**Comment by sv3k** [comment]
I would say it's an essential feature to have. Otherwise you need to keep the [datatype list](http://www.postgresql.org/docs/current/static/datatype.html) in your head :)

**Comment by kpowick** [comment]
I strongly support this idea and cannot believe the feature doesn't already exist.  Besides having to remember all possible PGSQL types, it is extremely tedious to manually type the datatype name (correctly) for each column added.

**Comment by 1st** [comment]
Any information about plans of implementation?

**Comment by timstacey** [comment]
Bump!

**Comment by sebastian-fredriksson-bernholtz** [comment]
I'm confused. Maybe I'm missing something, but this still doesn't seem to be available and the ticket is still open. It was added to the v1.2 milestone, but v1.2 was released in 2017 without the feature(?). Maybe this should be revisited and updated/closed.

Thanks for the amazing work on this product! It's awesome.

**Comment by jakob** [comment]
@sebastian-fredriksson-bernholtz You are right, we never added this feature. It seems that I originally planned to add this feature in 1.2, but never updated the milestone when we decided not to include it. We don't use Github milestones for development any more, so those tags might be outdated.

We're aware that this is an importannt missing feature, and we talk about it regularly, but it's never made it to the top of our to do list. We plan to add it in Postico 2, but I can't make any promises regarding timeline.
